<launch>
    <node pkg="gp_wind_regression" type="gp_wind_regression" name="gp_wind_regression_y" output="screen" />

	<param name="switch_xyz" type="int" value="1" /> <!-- Paramter to switch the regression between x,y,z. x=0, y=1, z=2.-->
	<param name="prediction_init_pub_topic" type="string" value="/gp_disturb_reg/predInit/y" /> <!-- Parameter to specfy topic name for for predInit.-->
	<param name="mean_pub_topic" type="string" value="/gp_disturb_reg/mu/y" /> <!-- Parameter to specfy topic name for mean.-->
	<param name="variance_pub_topic" type="string" value="/gp_disturb_reg/var/y" /> <!-- Parameter to specfy topic name for variance.-->
	<param name="mean_p_2std_dev_pub_topic" type="string" value="/gp_disturb_reg/mu_p_2std_dev/y" /> <!-- Parameter to specfy topic name for mean_p_2std_dev_pub_topic.-->
	<param name="mean_m_2std_dev_pub_topic" type="string" value="/gp_disturb_reg/mu_m_2std_dev/y" /> <!-- Parameter to specfy topic name for mean_m_2std_dev_pub_topic.-->
        <param name="feature_data_sub_topic" type="string" value="/dev/gp/feature_y_filtered" /> <!-- Parameter to specfy topic name for GP feature data. -->
	<param name="file_gp_initial" type="string" value="gp_initial_y.dat" /> <!-- Parameter to specfy file name for gp_initial.-->
	<param name="create_new_gp_initial" type="bool" value="true" /> <!-- Paramter to specify whether to create new gp_initial.-->
	<param name="use_gp_initial" type="bool" value="true" /> <!-- Paramter to use LSTM feature by utilizing gp_initial.-->
	<param name="use_sparse_gp_initial" type="bool" value="false" /> <!-- Paramter to use sparse_gp_initial.-->
	<param name="sizefactor_sparse_gp_initial" type="double" value="0.7" /> <!-- Paramter to set sizefactor for sparse_gp_initial.-->
	<param name="lambda" type="double" value="0.6" /> <!-- Paramter to set sizefactor for sparse_gp_initial.-->
	<!-- Load commmon parameters. -->
	<include file="$(find gp_wind_regression)/launch/common_params.xml" />
	
</launch>
